tests/comparisons/Filter/testDefault.php

```diff
@@ -0,0 +1,19 @@
+<?php
+declare(strict_types=1);
+
+namespace Search\Test\TestApp\Model\Filter;
+
+use Search\Model\Filter\FilterCollection;
+
+class PostsFilterCollection extends FilterCollection
+{
+    /**
+     * @return void
+     */
+    public function initialize(): void
+    {
+        $this->value('author_id');
```

Comment:
Would be nice if instead of repeating `$this` on each line method chaining was used similar to https://github.com/cakephp/bake/blob/85be666e8fbcc4a67ff0c0c7a66221623793cd5a/tests/comparisons/Model/testBakeTableValidation.php#L51-L53

Reply:
Yeah, for simplicity of the command, I decided not to do this.
Do you have an idea on how to easily achieve that?

For me, it seems enough as is, since it works both equally, and the lines don't increase.
So I am fine either way.

Reply:
Yes it's the same number of lines but avoid repeatation of `$this` would be nice. 

You can check bake's temple for tables and see how it generates the chaining. It should be just a matter of tweaking a loop but if it's too much trouble we can keep as it is.